app/src/main/java/com/github/polypoly/app/WelcomeActivity.kt

```diff
@@ -0,0 +1,134 @@
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.github.polypoly.app.menu.JoinGroupActivity
+import com.github.polypoly.app.menu.ProfileActivity
+import com.github.polypoly.app.menu.SettingsActivity
+import com.github.polypoly.app.menu.MenuComposable
+import com.github.polypoly.app.menu.kotlin.GameMusic
+
+import com.github.polypoly.app.ui.theme.PolypolyTheme
+
+/**
+ * This activity is the view that a player will see when launching the app, the idea is that
+ * this screen represents the "hub" from where all the main actions are made.
+ *
+ * These actions may be: creating a game, joining a game, logging in, settings, rules, leaderboards etc.
+ */
+class WelcomeActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent { WelcomeContent() }
+    }
+    @Preview(showBackground = true)
+    @Composable
+    fun WelcomePreview() { WelcomeContent() }
+
+    // ===================================================== MAIN CONTENT
+    @Composable
+    fun WelcomeContent() {
+        GameMusic.setSong(LocalContext.current, R.raw.mocksong)
+        GameMusic.startSong()
+        PolypolyTheme {
+            Surface(
+                modifier = Modifier.fillMaxSize(),
+                color = MaterialTheme.colors.background
+            ) {
+                Column(
+                    modifier = Modifier.fillMaxSize()
+                ) {
+                    // The first element is the logo of the game
+                    GameLogo()
+                    Spacer(modifier = Modifier.weight(1f))
+                    // Then the game buttons are in the center of the screen
+                    GameButtons()
+                    Spacer(modifier = Modifier.weight(1f))
+                    MenuComposable.RowButtons()
+                }
+            }
+        }
+    }
+
+
+
+    // ===================================================== WELCOME COMPONENTS
+    /**
+     * This composable is the main image of the game, the polypoly logo that'll be displayed
+     * in the welcome screen (i.e. Welcome Activity)
+     */
+    @Composable
+    fun GameLogo() {
+        Row(
+            horizontalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(20.dp)
```

Comment:
As I pointed it in the PR concern #23, I don't think have a constant for these values (they have 20.dp by coincidence) is a good idea, this would just move the values outside the composable and having it global would make it messy.
If we store each number in a companion object, we'll not know exactly what value affects what elements are here no values are common to more than one composable.

In the case were spacings are identical, I create a constant like in the line 30 of `MenuComposable`